docker/nginx/nginx-x509.conf

```diff
@@ -126,6 +126,12 @@ http {
             proxy_pass http://opcua-adapter:${MF_OPCUA_ADAPTER_HTTP_PORT};
         }
 
+        # Proxy pass to mainflux-twins
+        location /twins {
```

Comment:
location ~ ^/(twins|states) 

the states of the twins have to read from the twins service